src/Type/Enum/UserRoleEnum.php

```diff
@@ -20,7 +20,7 @@ public static function register_type() {
 		if ( ! empty( $editable_roles ) && is_array( $editable_roles ) ) {
 			foreach ( $editable_roles as $key => $role ) {
 
-				$formatted_role = WPEnumType::get_safe_name( $role['name'] );
+				$formatted_role = WPEnumType::get_safe_name( isset( $role['name'] ) ? $role['name'] : $key );
```

Comment:
Did you mean this to be: 

```php
$formatted_role = isset( $role['name'] ) ? WPEnumType::get_safe_name(  $role['name'] ) : WPEnumType::get_safe_name ( $key ) ;
```

Reply:
Both ways should return the same safe_name. I can change if it's more readable the other way.

Reply:
Oh wow. I totally read this as: 

```
$formatted_role = WPEnumType::get_safe_name( isset( $role['name'] ) ) ? $role['name'] : $key;
```

I missed the ternary _inside_ of `get_safe_name` 🤦‍♂️.

Reply:
I'm ok with it as-is. Just need me some reading glasses or something 🤓